lib/src/test/java/uk/gov/di/ipv/cri/passport/library/service/PassportSessionServiceTest.java

```diff
@@ -63,10 +63,42 @@ void shouldCreateSessionItem() throws ParseException {
 
         ArgumentCaptor<PassportSessionItem> passportSessionItemArgumentCaptor =
                 ArgumentCaptor.forClass(PassportSessionItem.class);
-        verify(dataStore).create(passportSessionItemArgumentCaptor.capture());
+        verify(mockDataStore).create(passportSessionItemArgumentCaptor.capture());
         assertNotNull(passportSessionItemArgumentCaptor.getValue().getCreationDateTime());
         assertEquals(
                 passportSessionItemArgumentCaptor.getValue().getPassportSessionId(),
                 passportSessionID);
     }
+
+    @Test
+    void shouldUpdateLatestDcsResponseResourceId() {
+        String passportSessionID = SecureTokenHelper.generate();
+        String latestDcsResponseResourceId = "test";
+
+        when(mockDataStore.getItem(passportSessionID)).thenReturn(new PassportSessionItem());
+        underTest.setLatestDcsResponseResourceId(passportSessionID, latestDcsResponseResourceId);
+
+        ArgumentCaptor<PassportSessionItem> passportSessionItemArgumentCaptor =
+                ArgumentCaptor.forClass(PassportSessionItem.class);
+        verify(mockDataStore).update(passportSessionItemArgumentCaptor.capture());
+        assertEquals(
```

Comment:
I think the same for this one as well, although weirdly sonar didn't pick it up so I'm not 100% sure

Reply:
Addressed in new PR.